cmd/compute-domain-daemon/computedomain.go

```diff
@@ -169,6 +169,11 @@ func (m *ComputeDomainManager) UpdateComputeDomainNodeInfo(ctx context.Context,
 	// Unconditionally update its IP address
 	nodeInfo.IPAddress = m.config.podIP
 
+	// Conditionally update its status
+	if newCD.Status.Status == "" {
+		newCD.Status.Status = nvapi.ComputeDomainStatusNotReady
+	}
```

Comment:
Thanks for that.

Can you explain why the `cd` object passed as function argument has an invalid (uninitialized?) Status?